helm/charts/values.yaml

```diff
@@ -0,0 +1,174 @@
+# Default values for cstor-operators.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+cspcOperator:
+  componentName: cspc-operator
+  poolManager:
+    image:
+      registry:
+      repository: openebs/cstor-pool-manager-amd64
+      tag: 2.0.0
+  cstorPool:
+    image:
+      registry:
+      repository: openebs/cstor-pool
+      tag: 2.0.0
+  cstorPoolExporter:
+    image:
+      registry:
+      repository: openebs/m-exporter
+      tag: 2.0.0
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
+    repository: openebs/cspc-operator-amd64
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 2.0.0
+  annotations: {}
+  resyncInterval: "30"
+  podAnnotations: {}
+  nodeSelector: {}
+  tolerations: []
```

Comment:
can we add resourcelimits tag as well , to configure the limits for the deployments templates